File: 1114.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift();};
var x = 1
while (x !=2002){
    x = parseInt(prompt("Digite a Senha"))
if(x != 2002){
    console.log("Senha Invalida")
}else{
    
    console.log("Acesso Permitido")
}
}

File: 1114.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int x = sc.nextInt();
        validarSenha(sc, x);
        sc.close();
    }

    public static void validarSenha(Scanner sc, int x) {
        if (x == 2002) {
            System.out.println("Acesso Permitido");
        } else {
            System.out.println("Senha Invalida");
            int novaSenha = sc.nextInt();
            validarSenha(sc, novaSenha);
        }
    }
}
